### ISSUE TITLE
Are priorities on minimize directives no longer supported?

### ISSUE THREAD

**Issue by adaml-ec** [potassco/clingcon#96]
clingcon 3 supported priorities on minimize statements via the @ operator.  For example:
```
&dom{0..10} = i.
&dom{0..10} = j.
&show {i}.
&show {j}.
&distinct{i; j}.
&minimize {i@2}.
&minimize {j@1}.
```

would produce i=0 j=1 since minimizing i had higher priority.

In clingcon 5 I get:
```
error: missing definition for operator:
  @
```

Was this feature removed?

It was also useful that I could use this to specify if a clingcon minimize directive should have higher or lower priority than a clingo `#minimize` in the same program.

**Comment by rkaminsk** [comment]
At the moment priorities are not supported. Currently, clingcon supports two minimization modes. One that translates to ASP's minimize constraint and one that handles the constraint natively. It would be relatively straightforward to support priorities even intermixing them with standard minimize constraints in the former one. The native one would be more involved and would probably not allow for mixing constraints.

I'll mark this as a feature request. Probably, no one is going to work on this anytime soon, though.

**Comment by adaml-ec** [comment]
Thanks for the response.  I had figured out about the translation that with `--translate-opt=max` I could get my `&minimize` treated as a priority 0 ASP minimize, and I could arrange my ASP priorities relative to that, but it is lacking a way to have more than one different priority within the clingcon `&minimize`.

We had been blocked on upgrading to clingcon 5 because we wanted the aspif support (https://github.com/potassco/clingcon/issues/67) but I saw that this is now supported, so thanks for that.  However we're also using the priorities so it looks like we may still be sticking with clingcon 3 until this is implemented.
